images/build/debian-iptables/distroless/Dockerfile

```diff
@@ -0,0 +1,64 @@
+# Copyright 2022 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+ARG GORUNNERIMAGE
+# Using directly debian base, as we are going to just copy the required binaries
+FROM debian:bullseye as build
+
+COPY stage-binary-and-deps.sh /
+COPY get-binaries-from-package.sh /
+COPY package-utils.sh /
+
+
+ARG STAGE_DIR="/opt/stage"
+RUN apt -y update && \
+    apt -y install bash curl && \
+    mkdir -p "${STAGE_DIR}" && \
+    /get-binaries-from-package.sh "${STAGE_DIR}" conntrack \
+    ebtables    \
+    ipset       \
+    iptables    \
+    kmod        && \
+    `# dash is the real Debian basic shell` \
+    /stage-binary-and-deps.sh /bin/dash "${STAGE_DIR}" && \
+    `# below binaries are used by iptables-wrapper-installer.sh` \
+    /stage-binary-and-deps.sh /bin/cat "${STAGE_DIR}" && \
+    /stage-binary-and-deps.sh /bin/chmod "${STAGE_DIR}" && \
+    /stage-binary-and-deps.sh /bin/grep "${STAGE_DIR}" && \
+    /stage-binary-and-deps.sh /bin/ln "${STAGE_DIR}" && \
+    /stage-binary-and-deps.sh /bin/rm "${STAGE_DIR}" && \
+    /stage-binary-and-deps.sh /usr/bin/wc "${STAGE_DIR}"
+
+RUN curl -o "${STAGE_DIR}"/iptables-wrapper-installer.sh https://raw.githubusercontent.com/kubernetes-sigs/iptables-wrappers/v2/iptables-wrapper-installer.sh && \
```

Comment:
Not that I expect the v2 tag to move, but
```
https://raw.githubusercontent.com/kubernetes-sigs/iptables-wrappers/e139a115350974aac8a82ec4b815d2845f86997e/iptables-wrapper-installer.sh
```
would be the stable reference to this content.  Unless you actively want this target to be able to be moved with the `v2` tag, please consider using the stable reference instead.

Reply:
@danwinship was discussing this with @robscott and we were wondering if maybe we can stick with the tag (not a commit version) AND if it's worth to follow semver here?

Reply:
I'd say stick with the tag. What's the point in having tags if people aren't going to trust them.

I didn't use semver because I don't expect to have lots of versions but I guess maybe it could be useful if we need small bugfixes...

Reply:
staying with the tag then!